functions/imagemagick/test/index.test.js

```diff
@@ -31,40 +31,64 @@ requestRetry = requestRetry.defaults({
   retryDelay: 1000,
 });
 
-const BUCKET_NAME = process.env.FUNCTIONS_BUCKET;
-const {BLURRED_BUCKET_NAME} = process.env;
 
 const safeFileName = 'bicycle.jpg';
 const offensiveFileName = 'zombie.jpg';
-
+const BUCKET_NAME = process.env.FUNCTIONS_BUCKET;
+const {BLURRED_BUCKET_NAME} = process.env;
+const blurredBucket = storage.bucket(BLURRED_BUCKET_NAME);
 const cwd = path.join(__dirname, '..');
 
-const blurredBucket = storage.bucket(BLURRED_BUCKET_NAME);
+// Successfully generated images require cleanup.
+let cleanupRequired = false;
```

Comment:
The identifier `cleanupRequired` should be scoped within the `describe` closure rather than file-scoped.
Could we put the boolean next to `let startFF, stopFF;`?